Runtime/Properties/HighlightProperty.cs

```diff
@@ -15,16 +17,22 @@ public class HighlightProperty : BaseHighlightProperty
         /// </summary>
         public Color? CurrentHighlightColor { get; protected set; }
             
+        [Obsolete("Use 'objectHighlighter' instead.")]
```

Comment:
the objectHighlighter or the DefaultHighlighter?

Reply:
```suggestion
        [Obsolete("Use 'DefaultHighlighter' instead.")]
```